README.md

```diff
@@ -11,82 +12,87 @@ Logging from the following services is supported for both cases:
 * [RedShift](https://aws.amazon.com/redshift/)
 * [S3](https://aws.amazon.com/s3/)
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 5.1.0. Submit pull-requests to master branch.
```

Comment:
Should this be 4.1.0?

Reply:
We're on 5.x.x. now.  My practice has been to increment this to the latest version number, since this info is mostly used by people newly using the module.  If we want to indicate when the terraform 0.12 support first started, I think we should change the text here.

Reply:
I think this is great. Thanks.